markets/perps-market/contracts/storage/PerpsAccount.sol

```diff
@@ -103,6 +103,7 @@ library PerpsAccount {
         self.collateralAmounts[synthMarketId] += amountToAdd;
     }
 
+    // TODO: rename this maybe?  not really withdrawing collateral, just accounting
     function withdrawCollateral(
```

Comment:
The adding method is called `addCollateralAmount` so maybe `removeCollateralAmount`?